spec/teaspoon/configuration_spec.rb

```diff
@@ -149,6 +152,19 @@
 
   end
 
+  describe ".coverage_ignored_files" do
+
+    it "returns an empty list if nothing was set" do
+      expect(subject.coverage_ignored_files).to eq([])
+    end
+
+    it "returns an array of  if they were comma separated" do
```

Comment:
here's where that behavior is exhibited: https://github.com/modeset/teaspoon/blob/1ed8cfe34a4abe6375df619f1bb595b39749a2b7/lib/teaspoon/suite.rb#L41-41